tools/maint/gen_struct_info.py

```diff
@@ -242,10 +242,7 @@ def generate_cmd(js_file_path, src_file_path, cflags):
   # Compile the program.
   show('Compiling generated code...')
 
-  if any('libcxxabi' in f for f in cflags):
-    compiler = shared.EMXX
-  else:
-    compiler = shared.EMCC
+  compiler = shared.EMXX if any('libcxxabi' in f for f in cflags) else shared.EMCC
```

Comment:
Can we tell it to avoid this kind of thing?   I'm not big a fan of the single line if / else syntax.

Reply:
% [`ruff rule SIM108`](https://docs.astral.sh/ruff/rules/if-else-block-instead-of-if-exp/)